src/style.css

```diff
@@ -0,0 +1,21 @@
+* {
```

Comment:
`font` is an inherited style. No need to add it to `*`

Reply:
transferred to the body tag